bundle/manifests/3scale-operator.clusterserviceversion.yaml

```diff
@@ -515,9 +515,9 @@ spec:
                 - name: RELATED_IMAGE_SYSTEM_MEMCACHED
                   value: memcached:1.5
                 - name: RELATED_IMAGE_BACKEND_REDIS
-                  value: quay.io/centos7/redis-6-centos7:latest
+                  value: quay.io/centos7/redis-6-centos7:centos7
```

Comment:
You also need to change it here: https://github.com/3scale/3scale-operator/blob/master/config/manager/manager.yaml#L64 otherwise the tests will fail on pr

Reply:
also, this redis image is wrong, we should instead use: quay.io/fedora/redis-6